woocommerce/woocommerce
# Update/e2e test utils

### All Submissions:






### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/25955 .

Take the existing e2e utilities and change them to be a reusable package.

- Move utils to `utils` folder in `e2e` (to match the upcoming `e2e-env` for that package)
- Move a lot of the existing utils in `index.js` to `page-utils.js`
- Add a `package.json`

Updated existing e2e tests to use the e2e-utils package

- Updated existing test's internal dependencies 
- Updated `package.json`to use this package

### How to test the changes in this Pull Request:

- Make sure all tests pass in headless mode;
- Make sure all tests pass in -dev mode.

### Other information:

* [x ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x ] Have you written new tests for your changes, as applicable?
* [ x] Have you successfully run tests with your changes locally?



### Changelog entry

N/A for e2e
